build/components/package.config.tex

```diff
@@ -49,6 +49,11 @@
     \usepackage{lmodern}
 \fi
 
+\if\CEINR 0
+    \usepackage{parskip}
+    \setlength{\parskip}{6pt}
+\fi
+
```

Comment:
Wenn du:

```tex
    \setlength{\parskip}{6pt}
    \setlength{\parindent}{0cm}
```

machst, brauchst man kein extra package 😉 
Dann könnte man das auch in `setup.tex` tun.

Reply:
Du hast Recht, die zweite Zeile kann in die setup.tex, aber das Parskip-Package arbeitet angeblich intelligenter im Unterdrücken von Einrückungen: https://mirror.informatik.hs-fulda.de/tex-archive/macros/latex/contrib/parskip/parskip.pdf

Aber ich kann das auch gerne rausnehmen, ich habe noch keinen Unterschied feststellen können und kein Paket wäre auch im Sinne der Build Time

Reply:
Ah okay hmm.

Also, wenn du mir eine Situation zeigen kannst, in der die einfache Lösung schlechter, als die package Lösung ist, können wir auch gern die package Variante nehmen. Bisher war die einfache Lösung jedoch auch hier im Template der Standard, und das hat noch nie jemand bemerkt 😅

Reply:
Ich denke mittlerweile auch, der Zweizeiler ist ist beste Lösung, bis wirklich Probleme auftreten.

Oder wie es eine zynische KI mal formuliert hat: __the best solution is usually the easiest one__